homeassistant/components/honeywell/config_flow.py

```diff
@@ -42,12 +40,23 @@ async def async_step_reauth_confirm(
     ) -> FlowResult:
         """Confirm re-authentication with Honeywell."""
         errors: dict[str, str] = {}
-
+        assert self.entry is not None
+        reauth_schema = vol.Schema(
+            {
+                vol.Required(
+                    CONF_USERNAME, default=self.entry.data.get(CONF_USERNAME, "")
+                ): str,
+                vol.Required(
+                    CONF_PASSWORD, default=self.entry.data.get(CONF_PASSWORD, "")
+                ): str,
+            }
+        )
```

Comment:
Add this back to REAUTH_SCHEMA

Reply:
Without the default by the way, we're going to handle that in `self.add_suggested_values_to_schema`